3-enrich/scala-common-enrich/src/main/scala/com.snowplowanalytics.snowplow.enrich/common/enrichments/EnrichmentManager.scala

```diff
@@ -169,6 +171,7 @@ object EnrichmentManager {
           ("tr_tt"   , (CU.stringToDoublelike, "tr_total")),
           ("tr_tx"   , (CU.stringToDoublelike, "tr_tax")),
           ("tr_sh"   , (CU.stringToDoublelike, "tr_shipping")),
+          ("tr_cu"   , (CU.stringToDoublelike, "tr_currency")),
```

Comment:
Currency is a string like GBP, it's not something that should be parsed to a Double.